editor/block-mover/test/mover-label.js

```diff
@@ -1,11 +1,6 @@
 /**
  * External dependencies
  */
-import { expect } from 'chai';
-
-/**
- * Internal dependencies
- */
 import { getBlockMoverLabel, getMultiBlockMoverLabel } from '../mover-label';
```

Comment:
This section should now be titled `Internal dependencies`, not `External dependencies`.